_maps/map_files/generic/lavaland.dmm

```diff
@@ -2080,16 +2080,9 @@
 	},
 /area/mine/living_quarters)
 "en" = (
-/obj/structure/rack,
```

Comment:
Not sure why this stuff was removed. I repathed `lavaland_door` into `necropolis_gate` and that was the only thing I changed.